core/Archive/ArchiveInvalidator.php

```diff
@@ -130,7 +132,11 @@ private function buildRememberArchivedReportIdForSiteAndDate($idSite, $date)
     private function buildRememberArchivedReportIdProcessSafe($idSite, $date)
     {
         $id  = $this->buildRememberArchivedReportIdForSiteAndDate($idSite, $date);
-        $id .= '_' . getmypid();
+        if (Process::isMethodDisabled('getmypid')) {
+	        $id .= '_' . Common::getRandomString();
```

Comment:
Should we cache the value of `Common::getRandomString()` so the same value gets used next time (to simulate a real PID)?

Reply:
@diosmosis this method should be executed max once per request as the next time the option is set... could even always use getRandomInt in theory

Reply:
I see, makes sense 👍 will merge